### DISCUSSION TITLE
Suggestion: Brain/Neuron Cells For Neural Nets

### DISCUSSION THREAD

**Discussion by Nogardagem** [MaxRobinsonTheGreat/EvolutionSimulatorV2#99] [Ideas and Suggestions]
As stated in a recent video, there was the idea of adding a neural network to the way that the creatures move, and I thought that it would make sense if there was a Neuron cell that allowed that to happen. Then, a creature would need both a neuron and an eye to get a neural network, instead of whatever brain it has now. This may change the scale of creatures, so I'm not exactly sure that it would fit in, but I think it could work out.

**Comment by theomgdev** [comment]
https://bboettcher3.github.io/LifeEngine/

Such a mod has already been made, but Max is sensitive about performance and wants to keep things simpler.

**Reply by theomgdev** [reply]
But organisms have only few neurons and because of that They are not able to evolve complex behavior.

**Comment by aaron-jencks** [comment]
I'm working on something like this, I'm going to port the game to a different language where speed won't be an issue
